ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -284,10 +290,6 @@ - (NSDictionary *)prepareDataForLayoutAnimatingWorklet:(NSMutableDictionary *)cu
   preparedData[@"targetGlobalOriginY"] = targetValues[@"globalOriginY"];
   preparedData[@"windowWidth"] = currentValues[@"windowWidth"];
   preparedData[@"windowHeight"] = currentValues[@"windowHeight"];
-  if (currentValues[@"transformMatrix"] != nil && targetValues[@"transformMatrix"] != nil) {
-    preparedData[@"currentTransformMatrix"] = currentValues[@"transformMatrix"];
-    preparedData[@"targetTransformMatrix"] = targetValues[@"transformMatrix"];
-  }
```

Comment:
Why did you move `transform` from Layout Animations to SETs? Does it not work with LAs?

Reply:
I don't want to change the legacy implementation of LayoutAnimations. Larger snapshots can negative influent on Layout Animation performance. At this moment, I want to add those new values only for SET's snapshot and test if it works properly.